Xwt/Xwt/FileSelector.cs

```diff
@@ -169,10 +169,11 @@ public DefaultFileSelectorBackend ()
 			var box = new HBox ();
 			box.Accessible.IsAccessible = true;
 			box.Accessible.Role = Accessibility.Role.Group;
-			box.Accessible.Title = Application.TranslationCatalog.GetString("File Selector");
+			box.Accessible.Description = Application.TranslationCatalog.GetString("Enter File Selector");
```

Comment:
@GouriKumari  What does "Enter" mean in this context? It's a description of the group, so VO will say "Enter File Selector, Group, to enter the group press ...", isn't the word "Enter" redundant here? It's obvious that the user needs to enter the group. The description is supposed to describe the purpose of the element, not how to interact with it.